package.json

```diff
@@ -10,10 +10,18 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+<<<<<<< HEAD
 
     "express": "^4.18.2",
 
     "sequelize": "^6.31.1"
 
+=======
```

Comment:
this requires fixing - the second set of dependencies here is unnecessary

Reply:
alright

Reply:
update changes